test/test_source.rb

```diff
@@ -43,11 +46,11 @@ def test_parsing
         "
       )
       list = PDD::VerboseSource.new(file, PDD::Source.new(file, 'hey')).puzzles
-      assert_equal 2, list.size
+      assert_equal 3, list.size
       puzzle = list.first
       assert_equal '2-3', puzzle.props[:lines]
-      assert_equal 'привет, how are you doing?', puzzle.props[:body]
-      assert_equal '44', puzzle.props[:ticket]
```

Comment:
@mbao01 why these two lines are deleted? The puzzle seem to be in place.

Reply:
I tried to add a puzzle that tests this new implementation of leading spaces. I think I should just append that rather than replace.